pyomo/solvers/plugins/converter/pico.py

```diff
@@ -72,7 +71,7 @@ def apply(self, *args, **kwargs):
                     raise ConverterError("File "+item+" does not exist!")
                 cmd = cmd + " "+item
         print("Running command: "+cmd)
-        pyutilib.subprocess.run(cmd)
+        subprocess.run(cmd)
```

Comment:
Should this have `stdout=subprocess.DEVNULL, stderr=DEVNULL`, because the default behavior of the `pyutilib.subprocess.run` was to suppress the command output?